### PR TITLE
add: Flat look for Haiku

### PR DESCRIPTION
**Pull request by unarix**
This package contains the FlatDecorator and FlatControlLook with three themes. This is the first version, tested in 64 and 32 bits architecture.

### CODE REVIEW

File: haiku-misc/haiku-flatstyle/haiku_flatstyle-0.2.recipe [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+SOURCE_DIR="haiku_darkstyle-$portVersion"
+
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86_gcc2 x86"
+
+USER_SETTINGS_FILES=""
+
+PROVIDES="
+	haiku_flatstyle$secondaryArchSuffix = $portVersion
+	FlatControlLook$secondaryArchSuffix = $portVersion
```
Comment (extrowerk) [inline haiku-misc/haiku-flatstyle/haiku_flatstyle-0.2.recipe:32]: addon: FlatControlLook$secondaryArchSuffix = $portVersion

File: haiku-misc/haiku-flatstyle/haiku_flatstyle-0.2.recipe [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+	FlatDecorator$secondaryArchSuffix = $portVersion
```
Comment (extrowerk) [inline haiku-misc/haiku-flatstyle/haiku_flatstyle-0.2.recipe:33]: addon: FlatDecorator$secondaryArchSuffix = $portVersion

File: haiku-misc/haiku-flatstyle/haiku_flatstyle-0.2.recipe [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+the computer or quit 'Deskbar' and 'Tracker' services from 'ProcessControler' (they \
+will restart auto).
+
+Credits for:
+- Nahuel Tello (nhtello)
+- Haiku DefaultDecorator and DefaultControlLook.
+"
+HOMEPAGE="https://github.com/unarix/haiku_darkstyle/"
+COPYRIGHT="2020, Nahuel Tello, Haiku Inc."
+
```
Comment (extrowerk) [inline haiku-misc/haiku-flatstyle/haiku_flatstyle-0.2.recipe:16]: Pls remove this line

File: haiku-misc/haiku-flatstyle/haiku_flatstyle-0.2.recipe [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+LICENSE="MIT
+		Zlib"
```
Comment (extrowerk) [inline haiku-misc/haiku-flatstyle/haiku_flatstyle-0.2.recipe:18]: 1 tab should be enough

File: haiku-misc/haiku-flatstyle/haiku_flatstyle-0.2.recipe [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+REVISION="1"
+SOURCE_URI="https://github.com/unarix/haiku_darkstyle/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="265ee577f31e723dedd3443df1d7c76f5d83b2134b712d8eb0377becc573898e"
+SOURCE_FILENAME="haiku_darkstyle-$portVersion.tar.gz"
+SOURCE_DIR="haiku_darkstyle-$portVersion"
+
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86_gcc2 x86"
+
+USER_SETTINGS_FILES=""
```
Comment (extrowerk) [inline haiku-misc/haiku-flatstyle/haiku_flatstyle-0.2.recipe:28]: No need for this.

File: haiku-misc/haiku-flatstyle/haiku_flatstyle-0.2.recipe [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+{
+	control_lookDir=$addOnsDir/control_look
+	mkdir -p $control_lookDir
+	cp -a FlatControlLook/objects/FlatControlLook $control_lookDir
+
+	decoratorsDir=$addOnsDir/decorators
+	mkdir -p $decoratorsDir
+	cp -a FlatDecorator/objects/FlatDecorator $decoratorsDir
+
+	dataD=$dataDir/UIThemes
```
Comment (extrowerk) [inline haiku-misc/haiku-flatstyle/haiku_flatstyle-0.2.recipe:63]: Is this for theme manager

File: haiku-misc/haiku-flatstyle/haiku_flatstyle-0.2.recipe [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+LICENSE="MIT
+		Zlib"
+REVISION="1"
+SOURCE_URI="https://github.com/unarix/haiku_darkstyle/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="265ee577f31e723dedd3443df1d7c76f5d83b2134b712d8eb0377becc573898e"
+SOURCE_FILENAME="haiku_darkstyle-$portVersion.tar.gz"
+SOURCE_DIR="haiku_darkstyle-$portVersion"
+
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86_gcc2 x86"
```
Comment (extrowerk) [inline haiku-misc/haiku-flatstyle/haiku_flatstyle-0.2.recipe:26]: I think no secondary arch dupport needed, AFAIK app_server runs always on primary arch. @somebody?

File: haiku-misc/haiku-flatstyle/haiku_flatstyle-0.2.recipe [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+from haiku-depot. After install it, select from the list: 'DarkFlat'/'LightFlat'/'MoonFlat'.
+
+Important: the UI needs to repaint all after select one theme, please restart \
+the computer or quit 'Deskbar' and 'Tracker' services from 'ProcessControler' (they \
+will restart auto).
+
+Credits for:
+- Nahuel Tello (nhtello)
+- Haiku DefaultDecorator and DefaultControlLook.
+"
```
Comment (extrowerk) [inline haiku-misc/haiku-flatstyle/haiku_flatstyle-0.2.recipe:13]: You can move " to the previous line.

Comment (extrowerk) [reply haiku-misc/haiku-flatstyle/haiku_flatstyle-0.2.recipe:13]: But it is not significant.

**Comment by extrowerk** [comment]
Untested by me, but looks ok.

**Comment by pulkomandy** [comment]
Is it ok to call the package "haiku"? It's not an official Haiku thing (yet?) so that could be misleading?

**Comment by unarix** [comment]
I dont have any problem to raname that; what do you suggest? its important that the name reflect what the package supposed to do... like "flat-UI" / "flat-style"

**Comment by pulkomandy** [comment]
Yes, I think flatstyle or flat-style is a good name. No need to be very creative here :)

File: haiku-misc/haiku-flatstyle/haiku_flatstyle-0.2.recipe (hunk 8 above)
Comment (unarix) [reply haiku-misc/haiku-flatstyle/haiku_flatstyle-0.2.recipe]: done!

**Comment by unarix** [comment]
done! the new nme is "flatstyle"

**Comment by unarix** [comment]
Hello all!! i need to make some change more? its ready now to merge? thnks!

**Pull request merged**